solutions/devsprint-bruno-ramos-3/DeliveryApp/Models/DeliveryTime.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DeliveryTime.swift
+//  DeliveryApp
+//
+//  Created by Amaryllis Baldrez on 17/02/22.
+//
+
+import Foundation
+
+struct DeliveryTime {
+    let min: Int
+    let max: Int
+    
+    init(from response: DeliveryTime) {
```

Comment:
Nesse caso o seu response é o DeliveryTime do response, certo? 
```suggestion
    init(from response: RestaurantResponse.DeliveryTime) {
```